Makefile

```diff
@@ -26,5 +26,9 @@ run: sourir
 clean:
 	ocamlbuild -clean
 
+install-deps:
```

Comment:
perfect!
I left FIXMEs in the readme.  can you  insert the "make install-deps" there